mslib/mswms/wms.py

```diff
@@ -517,9 +526,10 @@ def application():
         server_url = urllib.parse.urljoin(url, urllib.parse.urlparse(url).path)
 
         if (request_type in ('getcapabilities', 'capabilities') and
-                request_service == 'wms' and request_version in ('1.1.1', '')):
-            return_data, return_format = server.get_capabilities(server_url)
-        elif request_type in ('getmap', 'getvsec') and request_version in ('1.1.1', ''):
+                request_service == 'wms' and request_version in ('1.1.1', '1.3.0', '')):
+            return_data, return_format = server.get_capabilities(server_url,
+                                                                 request_version if request_version != "" else "1.3.0")
```

Comment:
would it be fine to have 1.3.0 as default in request_version above?

Reply:
Thinking more about it, I changed it to default to 1.1.1. Most other WMS Servers default to 1.1.1, too.
This will cause the least complications.